### PR TITLE
add pgx library

### PR DESCRIPTION
**Pull request by mhamrle**
part of https://github.com/monetas/gotary/issues/69

### CODE REVIEW

File: golang-base/install.sh [hunk 1]
```diff
@@ -2,12 +2,9 @@
 
 ### Setup various packages.
 apt-get update
-apt-get install -y --no-install-recommends software-properties-common
-add-apt-repository ppa:monetas/opentxs
-apt-get update
 apt-get install -y --no-install-recommends \
-    libzmq3-dev \
-    pkg-config
+    software-properties-common \
```
Comment (toxeus) [inline golang-base/install.sh]: AFAIK this is only needed for `add-apt-repository`

Comment (mhamrle) [inline golang-base/install.sh]: great, let's remove it

Comment (mhamrle) [inline golang-base/install.sh]: `ca-certificates` is still needed by github dependencies

**Comment by toxeus** [comment]
:+1:

**Comment by benma** [comment]
:+1:

**Pull request merged**